Provisioner/BuildProvisioner.php

```diff
@@ -45,11 +42,11 @@ public function __construct(
     }
 
     /**
-     * @param BuildEvent $event
+     * @param Task $task
      */
-    public function onBuild(BuildEvent $event)
+    public function run(Task $task)
     {
-        $this->task = $event->getTask();
+        $this->task = $task;
```

Comment:
Why not move this to a constructor (since we're no longer using events...)

Reply:
Can't, the provisioners are autowired, could be moved to a setter... (in the core bundle)